src/commands/casing.cpp

```diff
@@ -113,12 +113,12 @@ void AOClient::cmdTestify(int argc, QStringList argv)
 
     AreaData *l_area = server->getAreaById(areaId());
     if (l_area->testimonyRecording() == AreaData::TestimonyRecording::RECORDING) {
-        sendServerMessage("Testimony recording is already in progress. Please stop it before starting a new one.");
+        sendServerMessage("Testimony recording is already in progress. Please stop it with /end before starting a new one.");
     }
     else {
         clearTestimony();
         l_area->setTestimonyRecording(AreaData::TestimonyRecording::RECORDING);
-        sendServerMessage("Started testimony recording.");
+        sendServerMessage("Started testimony recording. Your next IC message will be a title. Use /end to stop recording.");
```

Comment:
Server doesn't actually care who the message comes from
```suggestion
        sendServerMessage("Started testimony recording. The next IC message will be a title. Use /end to stop recording.");
```